packages/create-platformatic/src/create-package-json.mjs

```diff
@@ -7,7 +7,7 @@ const packageJsonTemplate = (addTSBuild = false) => (`\
 {
   "scripts": {
     ${addTSBuild
-? ` "start": "yarn clean && platformatic {type} start",
+? `"start": "npm run clean && platformatic {type} start",
```

Comment:
I think we forgot to update the test for this line at https://github.com/platformatic/platformatic/blob/34def5623ad870d20da678ed8e16dab8c6523a9a/packages/create-platformatic/test/create-package-json.test.mjs#L67

Reply:
+1

Reply:
fixed, good spot.